### ISSUE TITLE
SSL for Laverna + storage

### ISSUE THREAD

**Issue by petrk94** [Laverna/laverna#703]
Hello
I would like to know how to use laverna with SSL / HTTPS?
I have set up my domain with SSL to use HTTPS, but when I try to access laverna in xxx.de/laverna
than its using http and not https. But when I type https://xxx.de/laverna than it works too
How I can redirect it to the HTTPS version?

The other question, I have it selfhosted on my server, are the notes all stored in the mysql database?
So when Im backup the mysql database, and restore the information, than laverna is working with the "imported" notes without problems?

Thanks

**Comment by jekuno** [comment]
Regarding question 1: Use a `.htaccess` file to always redirect from http to https.

**Comment by petrk94** [comment]
@jekuno thanks for your information.
I will change it as soon as possible.
I have read about the storaging of laverna, that its all stored in the browser cache, but what happens when I make a clean up with CCleaner or similar? Than the files are lost, its not very practical ?

**Comment by jekuno** [comment]
I'm new to laverna but you can enable sync with Dropbox and RemoteStorage. Also there are standalone desktop apps for all OS which probably save data in a dedicated laverna folder on your file system.

**Comment by petrk94** [comment]
Yes I have seen it, but not tested it yet, whether it access the browser storage and make it persistent in the file system when the App is installed locally

**Comment by petrk94** [comment]
I have changed few entries, but it didnt helped.
What I need to change in .htaccess to redirect to https automatically?
